src/main/java/com/jakuszko/hexagonal/user/domain/User.java

```diff
@@ -0,0 +1,31 @@
+package com.jakuszko.hexagonal.user.domain;
+
+import com.jakuszko.hexagonal.user.domain.exceptions.ShortUserNameException;
+import com.jakuszko.hexagonal.user.domain.exceptions.UserNameContainNumbersException;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+@Builder
+@Getter
+@Setter
```

Comment:
z tym `@Setter` proponuje ostrożnie ;) Czy na pewno każdy powinien móc wywołać user.setName('foo') skoro wystawiamy obok metodę changeName(...) z dodatkową walidacją itp? Raczej nie. Generalnie w kwestii obiektów domenowych proponuję podejście że jak coś może być niewidoczne na zewnątrz to najlepiej to ukryć

Reply:
Ok